app/src/pages/inside/launchesPage/modals/launchImportModal/launchImportModal.jsx

```diff
@@ -162,7 +162,15 @@ export class LaunchImportModal extends Component {
       ),
     };
   };
-
+  get validFiles() {
```

Comment:
I think it's better to use functions instead of field getters:
- it will be harder to make a mistake in name (calling unexisting function will thow an exception)
- using functions for computed data is less confusing (when accessing field it's not clear if this field is readonly, or not)